android/canonical/app/src/main/java/com/google/samples/quickstart/canonical/MainActivity.kt

```diff
@@ -41,10 +41,10 @@ class MainActivity : AppCompatActivity() {
         }
 
         R.id.bottom_navigation_item_map -> {
-          mapFragment = MapFragment()
+          mapsFragment = MapsFragment()
```

Comment:
Why change to plural?

Reply:
Just delete previous MapFragment() placeholder class and create new one MapsFragment().